onnx/check-model-extdata-alignment.py

```diff
@@ -0,0 +1,150 @@
+import os
```

Comment:
Is this intended to be part of the PR? I don't understand its purpose. I am also concerned about the code for extracting zip files etc., which have triggered some vulnerability issues in the past. It may be better to avoid such utilities, unless they are important.

Reply:
Just an example test script @ivberg explained to me. I can take over the PR and formalize the implementation.